man/chage.1.xml

```diff
@@ -194,6 +194,7 @@
 	    Apply changes in the <replaceable>CHROOT_DIR</replaceable>
 	    directory and use the configuration files from the
 	    <replaceable>CHROOT_DIR</replaceable> directory.
+	    Only support absolute path.
```

Comment:
I meant in the manpage too - "Only absolute paths are supported."  If you prefer I can change it inline as I apply.

Reply:
I'm sorry for misunderstanding your meaning.
done